tests/ImageSharp.Tests/project.json

```diff
@@ -20,9 +20,24 @@
     }
```

Comment:
> they need to be cleaner, conform to a basic set of rules and naming standards and be instantly obvious what they are testing.

Agreed. There are many test classes being non-conform to the basic formatting rules of the library, and this is not good. I just wanted to warn, that simply copy-pasting the default StyleCop behavior to the tests would be harmful to the project. 

Shouldn't we open a discussion about this? Or as an alternative you can just be a Wise and Merciful Dictator, and everything should be OK :laughing: